CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Change Log - @itwin/insights-client
+## 0.8.1
```

Comment:
This is a breaking change. Should be minor bump.

Reply:
Bumped to 0.9.0.